### ISSUE TITLE
Transaction and personal_sign can not work at metamask 4.2.2

### ISSUE THREAD

**Issue by bamboo-zh** [pink-room/walletconnectkit-android#13]
Transaction and personal_sign can not work at metamask 4.2.2
Can not alert the bottom dialog

**Comment by Neo-glitch** [comment]
I noticed this too, any help please. Thank you

**Comment by souravsoftwareer** [comment]
Yes me too not getting popup for perform transaction and sign transaction.

**Comment by souravsoftwareer** [comment]
See video attached
https://drive.google.com/file/d/1chqo9uSNvu37O2PRvkG3YOUFAZwfKJ-S/view?usp=sharing

**Comment by souravsoftwareer** [comment]
Perform Transaction
val toAddress = toAddressView.text.toString()
            val value = valueView.text.toString()
            showMessage(toAddress)

            lifecycleScope.launch {
                runCatching {
                       walletConnectKit.performTransaction(toAddress, value)
//                       val message = "Hello"
//                       walletConnectKit.personalSign(message)
                    }
                    .onSuccess {
                        showMessage("Transaction done!")
                        Log.e("success","Transaction done!")
                    }
                    .onFailure {
                        Log.e("fail",it.message ?: it.toString())
                        showMessage(it.message ?: it.toString())
                    }
            }

**Comment by jaydipumaretiya** [comment]
> 

Please check the address that you are passed.

**Comment by souravsoftwareer** [comment]
@jaydipumaretiya this is not issue of address .Mine address is correct it is issue of meta version i have tried with version 4.1.1 it transfer eth but using version 4.2.2 or greater creates issue.

**Comment by wangyuan-dev** [comment]
MetaMask version 4.2.2 or greater
I have same issue, Transaction and personal_sign can not work

**Comment by hywelingham** [comment]
did anyone solve this?

**Comment by qm921** [comment]
> did anyone solve this?

The parameters you pass are officially used, and I can get the signature by filling in the parameters that are officially used.

**Comment by qm921** [comment]
> 有没有人解决这个问题？

The parameters you pass in for production, I can get the signature by filling in the parameters for production.

**Comment by q-Mony** [comment]
> did anyone solve this?

The parameters you pass in for production, I can get the signature by filling in the parameters for production.

**Comment by Basavaraj367** [comment]
> Transaction and personal_sign can not work at metamask 4.2.2 Can not alert the bottom dialog 

hi have you got solution for this if you got please rply thanks

**Comment by Basavaraj367** [comment]
> @jaydipumaretiya this is not issue of address .Mine address is correct it is issue of meta version i have tried with version 4.1.1 it transfer eth but using version 4.2.2 or greater creates issue.

hi have got solution for sign message if got please reply thanks

**Comment by worstkiller** [comment]
Hey, Guys not able to sign a personal msg please please help 🙏 🙏 .

Right now called a personal sign method from code its opening the wallet but dialog is not coming to confirm the person sign. Please with any code or something.
